config/etfuturum/blocksitems.cfg

```diff
@@ -0,0 +1,445 @@
+    #  [default: true]
+    B:enableLingeringPotions=true
+
+    #  [default: true]
+    B:enableMutton=true
+
+    #  [default: true]
+    B:enableNewDyes=true
+
+    # Appears in stronghold corridor and dungeon chests. [default: true]
+    B:enableOtherside=true
+
+    # Appears in Nether fortress chest loot. [default: true]
+    B:enablePigstep=true
+
+    # If true, vanilla and Et Futurum copper ores will drop raw ore items. [default: true]
+    B:enableRawOres=false
+
+    #  [default: true]
+    B:enableSuspiciousStew=true
+
+    #  [default: true]
+    B:enableTippedArrows=true
+
+    #  [default: true]
+    B:enableTotemUndying=true
+}
+
+
+##########################################################################################################
+# natural blocks
+#--------------------------------------------------------------------------------------------------------#
+# Blocks that can generate naturally in your world. Check world.cfg for generation values.
+##########################################################################################################
+
+"natural blocks" {
+    # Set how hard deepslate is, in case you like the look of deepslate but don't want to deal with the additional hardness. Regular stone is 1.5. [range: 0.0 ~ 100.0, default: 3.0]
+    S:deepslateHardness=2.3333333333
+
+    # Enables tinted glass, amethyst blocks, budding amethyst and amethyst crystals. Also enables the item too. [default: true]
+    B:enableAmethyst=false
+
+    # Enables bamboo wood and all of its wood subtypes, including the bamboo stalks themselves. [default: true]
+    B:enableBambooBlocks=false
+
+    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
+    B:enableBasalt=true
+
+    #  [default: true]
+    B:enableBlueIce=true
+
+    #  [default: true]
+    B:enableBoneBlock=true
+
+    #  [default: true]
+    B:enableCalcite=true
+
+    # Enables cherry wood and all of its wood subtypes. [default: true]
+    B:enableCherryBlocks=true
+
+    # Enables chorus plants and purpur blocks [default: true]
+    B:enableChorusBlocks=true
+
+    #  [default: true]
+    B:enableCoarseDirt=true
+
+    # Copper ore and copper blocks, variants, and waxed variants. (Slime balls are used if no mod introduces wax and if honey is disabled) [default: true]
+    B:enableCopper=false
+
+    #  [default: true]
+    B:enableCornflower=true
+
+    #  [default: true]
+    B:enableCryingObsidian=true
+
+    #  [default: true]
+    B:enableDeepslate=true
+
+    # Enable deepslate ores for copper ore and vanilla ores when deepslate generates over them. [default: true]
+    B:enableDeepslateOres=true
+
+    #  [default: true]
+    B:enableGrassPath=true
+
+    #  [default: true]
+    B:enableLilyOfTheValley=true
+
+    #  [default: true]
+    B:enableMagmaBlock=true
+
+    # Enables mud, packed mud and mud bricks, as well as the mud brick stairs, slabs and walls. [default: true]
+    B:enableMud=true
```

Comment:
we already have mud and mud bricks, stairs, slabs, and armor

Reply:
AFAICT, mud doesn't actually generate in the world. This just gives us access to some nice blocks that I'll have to create gregified recipes for.

Reply:
The textures are different. IMO the EFR textures are nicer. AFAICT, the EFR mud doesn't spawn in the world. I would like to make the recipes similar and use the biomes o plenty mud as the base crafting ingredient but have access to both options. Thoughts?

Reply:
there is very much mud worldgen code in the mod. And having all these items double would be weird.
maybe just make a texture pack if you prefer certain textures?

Reply:
SG. Disabled.